src/components/Root/Root.js

```diff
@@ -9,7 +9,8 @@ import { ApolloProvider } from 'react-apollo';
 import ErrorBoundary from '@folio/stripes-components/lib/ErrorBoundary';
 import { metadata } from 'stripes-config';
 
-import { RootContext } from './RootContext';
+import { ConnectContext } from '@folio/stripes-connect';
+// import { RootContext } from './RootContext';
```

Comment:
Either delete the commented-code altogether, or leave it in place BUT with a comment explaining what it did and why we don't do it that any longer.